drivers/hmis_external_apis/app/models/hmis_external_apis/ac_hmis/unit_availability_sync.rb

```diff
@@ -0,0 +1,34 @@
+###
+# Copyright 2016 - 2023 Green River Data Analysis, LLC
+#
+# License detail: https://github.com/greenriver/hmis-warehouse/blob/production/LICENSE.md
+###
+
+module HmisExternalApis::AcHmis
+  # Track local and synced changes
+  class UnitAvailabilitySync < ::HmisExternalApis::HmisExternalApisBase
+    self.table_name = 'hmis_external_unit_availability_syncs'
+    belongs_to :project, class_name: 'Hmis::Hud::Project'
+    belongs_to :unit_type, class_name: 'Hmis::UnitType'
+    belongs_to :user, class_name: 'Hmis::User'
+
+    scope :dirty, -> { where('local_version > synced_version') }
```

Comment:
For consistency, can we stick with Arel here?
```suggestion
    scope :dirty, -> { where(arel_table[:local_version].gt(arel_table[:synced_version])) }
```